packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -99,7 +106,7 @@ function MainLayoutHeader({
       <SearchPanel
         className={{
           container: classNames(
-            'left-0 top-0 z-header mx-auto bg-background-default py-3 tablet:left-16 laptop:left-0 laptop:bg-transparent',
+            'left-0 top-0 z-header mx-auto flex-1 items-center bg-background-default py-3 tablet:left-16 laptop:left-0 laptop:bg-transparent',
```

Comment:
We'll just have to make sure this items-center doesn't add edge case for any of the renders inside. (mobile/desktop)

Reply:
Switch to make the search panel smaller on laptop screens and added another breakpoint for laptopL. That way, we do not need to make changes in the layout here.